### PR TITLE
clear-containers-image: Update release, Add changes to changelog

### PR DESCRIPTION
**Pull request by gorozco1**
* Update release every udpate
     * Add changes on image with scripts/get-image-changes.sh on the
     debian.changelog

    Fixes: #152
    Fixes: #153

### CODE REVIEW

File: clear-containers-image/update_image.sh [hunk 1]
```diff
@@ -6,37 +6,55 @@
...
+if [ $# -ne 1 ]; then
+    cat << EOT
+Usage:
+$0 <clear-linux-version>
+
+You need to provide the version to update the image
+EOT
+    exit
+fi
+script_dir=$(dirname "$0")
```
Comment (erick0z) [inline clear-containers-image/update_image.sh:18]: scripts_dir points to the same directory as cc_image_dir. Can we use just one variable?

**Review by erick0z — CHANGES_REQUESTED** [review]

File: clear-containers-image/update_image.sh (hunk 1 above)
Comment (gorozco1) [reply clear-containers-image/update_image.sh:18]: is not, `script_dir` could be `./`and `cc_image_dir` is `/home/foo/packaging/clear-containers-image`

Comment (erick0z) [reply clear-containers-image/update_image.sh:18]: I see, script_dir is useful when calling the script from any other directory, which is uncommon.

Comment (erick0z) [reply clear-containers-image/update_image.sh:18]: +1

**Review by erick0z — APPROVED** [review]

**Pull request merged**